tests/environment/docker-compose-all-amps-test.yml

```diff
@@ -47,6 +47,11 @@ services:
         -Ds3.bucketName=bamboo-s3-conn-default
         -Ds3.autoLowerCaseBucketName=true
 
+        -Dcontent.remote.default.contentRefHandler.source.s3.bucketRegion=us-east-1
```

Comment:
I assume this also work with empty values ? If so, we should also raise a tech debt JIRA to add default (empty) properties to MM Repo AMP.

Reply:
It is not working with empty values -> https://travis-ci.com/github/Alfresco/acs-packaging/jobs/334631259, so I assume we need these default configs.